test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -457,6 +462,72 @@ public void checkpointWriterInterleavedTest() throws Exception {
         }
     }
 
+    /** Test the CheckpointWriter write size limit.
+     *
+     * CheckpointWriter aggregates a batch of 50 SMREntries into one
```

Comment:
should this comment be updated to reflect the new behavior, i.e., 50 entries or a limit of 25MB?

Reply:
I will make that change